<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QWidget" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>131</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_favgenres">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Favourite genres:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_favourite_genres">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_favactors">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Favourite actors:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_favmovies">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Favourite movies:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_about">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>About me:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_about">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>151</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>131</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>131</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>131</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>210</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_edit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>24</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>241</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_genres">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>431</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_status">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
